dataland-document-manager/src/main/kotlin/org/dataland/documentmanager/controller/DocumentController.kt

```diff
@@ -47,4 +59,51 @@ class DocumentController(
             .header(HttpHeaders.CONTENT_TYPE, "${document.type.mediaType}")
             .body(documentContent)
     }
+
+    override fun patchDocumentMetaInfo(
+        documentId: String,
+        documentMetaInfoPatch: DocumentMetaInfoPatch,
+    ): ResponseEntity<DocumentMetaInfoResponse> {
+        if (documentMetaInfoPatch.isNullOrEmpty()) {
+            throw InvalidInputApiException(
+                summary = "DocumentMetaInfoPatch must not be null or empty.",
```

Comment:
Die Nachricht hier sollte weniger technisch formuliert sein. Das ist ja eine allgemeine message an den user der vielleicht nicht weiß wie die Variablen heißen. Also vielleicht eher sowas wie "At least one parameter has to be changed"